### PR TITLE
MMT-1611 Add data type version number

### PR DESCRIPTION
**Pull request by htranho**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -43,8 +43,11 @@ class Application < Rails::Application
 
     # Versions of UMM for the different metadata types MMT is on
     config.umm_c_version = 'vnd.nasa.cmr.umm+json; version=1.10'
```
Comment (htranho) [inline config/application.rb]: parse existing entry

File: app/views/layouts/_manage_nav.html.erb [hunk 2]
```diff
@@ -2,15 +2,19 @@
   <div class="content">
     <h2 class="<%= 'current' if current_manage_title == 'manage_collections' %>">
       <%= link_to 'Manage Collections', manage_collections_path %>
+      <sup><span class="eui-badge--sm" style="margin-left: 2px"><%= "#{Rails.configuration.umm_c_version_label}" %></span></sup>
```
Comment (htranho) [inline app/views/layouts/_manage_nav.html.erb]: use css class for the margin

File: spec/features/header/header_spec.rb [hunk 3]
```diff
@@ -14,6 +14,24 @@
       it 'has "Manage Collections" as the underlined current header link' do
         within 'main header' do
           expect(page).to have_css('h2.current', text: 'Manage Collections')
+          within 'h2.current' do
```
Comment (htranho) [inline spec/features/header/header_spec.rb]: Separated the test (different it)

File: spec/features/header/header_spec.rb [hunk 4]
```diff
@@ -14,6 +14,24 @@
...
+            expect(page).to have_css('span.eui-badge--sm', count: 1)
```
Comment (htranho) [inline spec/features/header/header_spec.rb]: Compare value to hardcoded value

File: app/helpers/manage_metadata_helper.rb [hunk 5]
```diff
@@ -42,4 +42,10 @@ def edsc_map_path
     edsc_root = service_configs['earthdata'][Rails.configuration.cmr_env]['edsc_root']
     "#{edsc_root}/search/map"
   end
+
+  def display_version(mime_type)
+    index = mime_type.index('version=')
+    version = mime_type[(index + 8)..-1]
```
Comment (htranho) [inline app/helpers/manage_metadata_helper.rb]: version variable not needed here

File: spec/features/header/header_spec.rb [hunk 6]
```diff
@@ -16,6 +16,32 @@
           expect(page).to have_css('h2.current', text: 'Manage Collections')
         end
       end
+
+      it 'has version label' do
+        within 'main header h2.current' do
+          expect(page).to have_css('span.eui-badge--sm.umm-version-label')
+          expect(page).to have_content('1.10')
```
Comment (htranho) [inline spec/features/header/header_spec.rb]: set config before test

**Comment by jceaser** [comment]
just as a note, despite what Bamboo may say, this will go into release 1.89.x

**Pull request merged**